src/mjolnir/edgeinfobuilder.cc

```diff
@@ -34,7 +34,9 @@ void EdgeInfoBuilder::set_bike_network(const uint32_t bike_network) {
 
 // Sets the speed limit in KPH.
 void EdgeInfoBuilder::set_speed_limit(const uint32_t speed_limit) {
-  if (speed_limit > kMaxSpeedKph) {
+  if (speed_limit == kUnlimitedSpeedLimit) {
```

Comment:
i guess you can combine this if with the else below since they do the same thing. perhaps you did this to make it more obvious? why not just `way.s = s < max || s == unlim ? s : max_s`. dont feel strongly about it, feel free to dismiss